src/routes/safe/components/Apps/index.tsx

```diff
@@ -43,12 +44,13 @@ const operations = {
 }
 
 function Apps({ closeModal, closeSnackbar, enqueueSnackbar, openModal }) {
-  const [appList, setAppList] = useState([])
+  const [appList, setAppList] = useState<Array<SafeApp>>([])
   const [legalDisclaimerAccepted, setLegalDisclaimerAccepted] = useState(false)
-  const [selectedApp, setSelectedApp] = useState()
+  const [selectedApp, setSelectedApp] = useState<string>()
   const [loading, setLoading] = useState(true)
   const [appIsLoading, setAppIsLoading] = useState(true)
-  const [iframeEl, setIframeEl] = useState(null)
+  const [iframeEl, setIframeEl] = useState<HTMLIFrameElement>(null)
```

Comment:
same here

Reply:
Are you sure? Typescript should infer the type

Reply:
https://codewithstyle.info/Using-React-useState-hook-with-TypeScript/

`const [pill, setPill] = useState<'red' \| 'blue'>();`

Note that the inferred type of pill is now "red" | "blue" | undefined (because this piece of state is initially empty).

Reply:
<img width="1428" alt="Screenshot 2020-05-27 at 17 59 45" src="https://user-images.githubusercontent.com/16622558/83029594-e137b980-a043-11ea-9ae1-2e799010ff2a.png">

Reply:
looks like it only adds undefined type if you don't provide an initial value

Reply:
weird, my editor does not complain about that. If I run `yarn ts` on my local, it does not complain either:

```
✔ ~/develop/safe-react [issue-686|⚑ 11] 
11:09 $ yarn tsc
yarn run v1.22.4
$ /home/nicosampler/develop/safe-react/node_modules/.bin/tsc
Done in 8.34s.
``` 
Could it be a problem with your editor? 

> looks like it only adds undefined type if you don't provide an initial value
That's correct. 

I'll change it anyway, as we are initializing it as null.

Reply:
because we are not using typescript's compiler in strict mode

Reply:
screenshot was done in a project where typescript compiler is in strict mode

Reply:
when we're writing/fixing our code we should imagine that TS compiler is in strict mode, because at some point in the future our goal is to enable it